Makefile

```
@@ -0,0 +1,24 @@
+install:
+	cargo install --path crates/solidity && \
+	npm install && npm fund
+
+test:
+	cargo install --path crates/solidity && \
+	npm install && npm fund && \
+	cargo test --manifest-path crates/solidity/Cargo.toml 
+	npm run test:cli
+
+test-solidity:
+	cargo test --manifest-path crates/solidity/Cargo.toml
+
+test-cli:
+	npm run test:cli
+
+clean:
+	cargo clean && \
+	rm -rf node_modules && \
+	rm -rf crates/solidity/src/tests/cli-tests/artifacts && \
+	rm -f ~/.cargo/bin/zksolc && \
+	rm -f package-lock.json
+
+
```

```suggestion
.PHONY : install test test-solidity test-cli test-integration clean

install :
        cargo install --path crates/solidity && \
        npm install && npm fund

test : install test-integration test-cli test-solidity

test-integration : install
        cargo test --package revive-integration

test-solidity : install
        cargo test --package revive-solidity

test-cli : install
        npm run test:cli

clean :
        cargo clean && \
        rm -rf node_modules && \
        rm -rf crates/solidity/src/tests/cli-tests/artifacts && \
        rm -f ~/.cargo/bin/zksolc && \
        rm -f package-lock.json
 
```

We should call the targets directly this way it's much more maintainable :) Also we'll add more in the future (more tests. I'm about to integrate a benchmarking suite etc..) and I think it'd make sense to use the Makefile for those as well :)

---

@xermicus 
Commit add to the PR